depends/packages/curl.mk

```diff
@@ -1,9 +1,9 @@
 package=curl
 GCCFLAGS?=
-$(package)_version=7.71.1
+$(package)_version=7.74.0
 $(package)_download_path=https://curl.haxx.se/download/
 $(package)_file_name=$(package)-$($(package)_version).tar.gz
-$(package)_sha256_hash=59ef1f73070de67b87032c72ee6037cedae71dcb1d7ef2d7f59487704aec069d
+$(package)_sha256_hash=E56B3921EEB7A2951959C02DB0912B5FCD5FDBA5ACA071DA819E1ACCF338BBD7
```

Comment:
Seems the hash doesn't match for curl:

```
Fetching curl-7.74.0.tar.gz from https://curl.haxx.se/download/
/home/runner/work/Gridcoin-Research/Gridcoin-Research/depends/work/download/curl-7.74.0/curl-7.74.0.tar.gz.temp: FAILED
sha256sum: WARNING: 1 of 1 computed checksums did NOT match
```

Might be because it's all caps.

Reply:
Yeah, the BusyBox `sha256sum` used for the ARM build is picky about casing. The GNU version doesn't care.

Reply:
```suggestion
$(package)_sha256_hash=e56b3921eeb7a2951959c02db0912b5fcd5fdba5aca071da819e1accf338bbd7
```